app/components/windows/Main.vue.ts

```diff
@@ -158,6 +158,8 @@ export default class Main extends Vue {
 
   created() {
     window.addEventListener('resize', this.windowSizeHandler);
+
+    document.getElementsByTagName('body')[0].classList.add(this.theme);
```

Comment:
Why does this need to be on `body` now?  I'm not a huge fan of this or the replacement we're doing in the customization service.

Reply:
this isn't actually necessary, was an artifact from when i was trying out some things; will remove it